### PR TITLE
docs: Add docstring to all methods

### PR DESCRIPTION
**Pull request by duwenxin99**
add docstring to all methods

### CODE REVIEW

File: src/langchain_google_alloydb_pg/chat_message_history.py [hunk 1]
```diff
@@ -51,6 +51,18 @@ def __init__(
         table_name: str,
         messages: List[BaseMessage],
     ):
+        """AlloyDBChatMessageHistory constructor.
+
+        Args:
+            key (object): Key to prevent direct constructor usage.
+            engine (AlloyDBEngine): AlloyDB engine to use.
```
Comment (averikitsch) [inline src/langchain_google_alloydb_pg/chat_message_history.py:58]:
```suggestion
            engine (AlloyDBEngine): database connection pool.
```

File: src/langchain_google_alloydb_pg/engine.py [hunk 2]
```diff
@@ -111,6 +118,17 @@ def __init__(
         loop: Optional[asyncio.AbstractEventLoop],
         thread: Optional[Thread],
     ) -> None:
+        """AlloyDBEngine constructor.
+
+        Args:
+            key(object): Prevent direct constructor usage.
+            engine(AsyncEngine): Async engine to create AlloyDBEngine from.
```
Comment (averikitsch) [inline src/langchain_google_alloydb_pg/engine.py:125]:
```suggestion
            engine(AsyncEngine): Async engine connection pool.
```

File: src/langchain_google_alloydb_pg/engine.py [hunk 3]
```diff
@@ -235,6 +307,7 @@ async def afrom_instance(
 
     @classmethod
     def from_engine(cls: Type[AlloyDBEngine], engine: AsyncEngine) -> AlloyDBEngine:
+        """Create an AlloyDBEngine instance from engine."""
```
Comment (averikitsch) [inline src/langchain_google_alloydb_pg/engine.py:310]:
```suggestion
        """Create an AlloyDBEngine instance from an AsyncEngine."""
```

File: src/langchain_google_alloydb_pg/engine.py [hunk 4]
```diff
@@ -352,6 +452,7 @@ def init_vectorstore_table(
         )
 
     async def ainit_chat_history_table(self, table_name: str) -> None:
+        """Create a new chat history table."""
```
Comment (averikitsch) [inline src/langchain_google_alloydb_pg/engine.py:455]: Can you expand this to be similar to `init_vectorstore_table`

**Review by averikitsch — APPROVED** [review]
Approved, but please see comments.

File: src/langchain_google_alloydb_pg/engine.py (hunk 4 above)
Comment (duwenxin99) [reply src/langchain_google_alloydb_pg/engine.py]: expanded

**Pull request merged**